app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/JourneyStatus.scala

```diff
@@ -59,7 +60,9 @@ object JourneyStatus {
 
     final case object IndividualWantsToRegisterTrust extends RegistrationStatus
 
-    final case class IndividualSupplyingInformation(name: Option[Name]) extends RegistrationStatus
+    final case class IndividualSupplyingInformation(name: Option[Name], address: Option[Address]) extends RegistrationStatus
```

Comment:
Would you mind changing this to IndividualWithCleanSkin? I know it sounds terrible but that is the common term that HMRC use to refer to such an individual.